private-templates-service/client/public/ACDesigner/containers/adaptivecards-defaulthost.css

```diff
@@ -0,0 +1,205 @@
+.card-frame {
+    border: 1px solid #F1F1F1;
+    box-shadow: 0 0 15px -5px rgba(0, 0, 0, 0.4);
+    background: white;
+}
+
+.ac-media-poster {}
```

Comment:
necessary?

Reply:
This file, along with all the CSS files in this PR are simply copied from the adaptivecards-designer node_module, so I haven't modified these files